### ISSUE TITLE
The Size of Image for Pre-trained Model

### ISSUE THREAD

**Issue by psj-jerry** [phoenixtreesky7/CFEN-ViT-Dehazing#10]
Hi, I would like to test my own test set using the test code given in your paper, but the image size given in your paper requires 512*512, can this size be changed? Also your training method is not given in detail, there is no way to reproduce it, how can I solve this? I look forward to your reply, thank you very much!
